commit 8d3243830b69697c7ebe7f6523a8372d9638403a
Author: Michael Zoran <mzoran@crowfest.net>
Date:   Thu Jan 12 19:10:07 2017 -0800

    ARM64: Enable HDMI audio and vc04_services in bcmrpi3_defconfig
    
    Signed-off-by: Michael Zoran <mzoran@crowfest.net>

diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index c7e891d72969..4b90f9b64abe 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -841,6 +841,7 @@ CONFIG_SND_VIRMIDI=m
 CONFIG_SND_MTPAV=m
 CONFIG_SND_SERIAL_U16550=m
 CONFIG_SND_MPU401=m
+CONFIG_SND_BCM2835=m
 CONFIG_SND_USB_AUDIO=m
 CONFIG_SND_USB_UA101=m
 CONFIG_SND_USB_CAIAQ=m
@@ -1125,7 +1126,6 @@ CONFIG_FB_TFT_UPD161704=m
 CONFIG_FB_TFT_WATTEROTT=m
 CONFIG_FB_FLEX=m
 CONFIG_FB_TFT_FBTFT_DEVICE=m
-# CONFIG_BCM2708_VCHIQ is not set
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1104_8d3243830b69697c7ebe7f6523a8372d9638403a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1104_8d3243830b69697c7ebe7f6523a8372d9638403a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
